webspark.profile

```diff
@@ -45,17 +73,9 @@ function webspark_form_install_configure_submit($form, FormStateInterface $form_
   }
 }
 
-function webspark_install_tasks_alter(&$tasks, $install_state) {
-  $academic_site = \Drupal::state()->get('academic_site', FALSE);
-  $tasks['webspark_install_configure_header_form'] = [
-    'display_name' => t('Header'),
-    'type' => 'form',
-    'function' => 'Drupal\webspark_installer_forms\Form\WebsparkConfigureHeaderForm',
-    'run' => INSTALL_TASK_RUN_IF_NOT_COMPLETED,
-  ];
-  $tasks['webspark_install_configure_ga_form'] = [
-    'display_name' => t('Google Analytics'),
-    'type' => 'form',
-    'function' => 'Drupal\webspark_installer_forms\Form\WebsparkConfigureGAForm',
-  ];
-}
+/**
+ * Implements hook_preprocess_template().
```

Comment:
Don't let this hold up the PR. It can merge. But perhaps this mis-matched hook implements note can be corrected in a future update.